samples/python/benchmark/sync_benchmark/README.md

```diff
@@ -0,0 +1,97 @@
+| Tensor Operations | [openvino.runtime.Tensor.get_shape], [openvino.runtime.Tensor.data] | Get a tensor shape and its data. |
+
+| Options | Values |
+| :--- | :--- |
+| Validated Models | [alexnet](@ref omz_models_model_alexnet), [googlenet-v1](@ref omz_models_model_googlenet_v1) [yolo-v3-tf](@ref omz_models_model_yolo_v3_tf), [face-detection-0200](@ref omz_models_model_face_detection_0200) |
+| Model Format | OpenVINO™ toolkit Intermediate Representation (\*.xml + \*.bin), ONNX (\*.onnx) |
+| Supported devices | [All](../../../../docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md) |
+| Other language realization | [C++](../../../cpp/benchmark/sync_benchmark/README.md) |
+
+## How It Works
+
+The sample compiles a model for a given device, randomly generates input data, performs synchronous inference multiple times for a given number of seconds. Then processes and reports performance results.
+
+You can see the explicit description of
+each sample step at [Integration Steps](../../../../docs/OV_Runtime_UG/integrate_with_your_application.md) section of "Integrate OpenVINO™ Runtime with Your Application" guide.
+
+## Building
+
+To build the sample, please use instructions available at [Build the Sample Applications](../../../../docs/OV_Runtime_UG/Samples_Overview.md) section in OpenVINO™ Toolkit Samples guide.
+
+## Running
+
+```
+python sync_benchmark.py <path_to_model>
+```
+
+To run the sample, you need specify a model:
+
+- you can use [public](@ref omz_models_group_public) or [Intel's](@ref omz_models_group_intel) pre-trained models from the Open Model Zoo. The models can be downloaded using the [Model Downloader](@ref omz_tools_downloader).
+
+> **NOTES**:
+>
+> - Before running the sample with a trained model, make sure the model is converted to the intermediate representation (IR) format (\*.xml + \*.bin) using the [Model Optimizer tool](../../../../docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md).
+>
+> - The sample accepts models in ONNX format (.onnx) that do not require preprocessing.
+
+### Example
+
+1. Install the `openvino-dev` Python package to use Open Model Zoo Tools:
+
+```
+python -m pip install openvino-dev[caffe,onnx,tensorflow2,pytorch,mxnet]
+```
+
+2. Download a pre-trained model using:
+
+```
+omz_downloader --name googlenet-v1
+```
+
+3. If a model is not in the IR or ONNX format, it must be converted. You can do this using the model converter:
+
+```
+omz_converter --name googlenet-v1
+```
+
+4. Perform benchmarking using the `googlenet-v1` model on a `CPU`:
+
+```
+python sync_benchmark.py googlenet-v1.xml
+```
+
+## Sample Output
+
+The application outputs performance results.
+
+```
+[ INFO ] OpenVINO:
+         API version............. <version>
+[ INFO ] Count:          1413 iterations
+[ INFO ] Duration:       15008.32 ms
+[ INFO ] Latency:
+[ INFO ]     Median:     10.45 ms
+[ INFO ]     AVG:        10.62 ms
+[ INFO ]     MIN:        8.79 ms
+[ INFO ]     MAX:        16.95 ms
```

Comment:
I'd consider updating these like so: AVG -> Average, MIN -> Min, MAX -> Max. This looks a bit less bold, aligns with the C++ version output, plus after benchmarks alignment python benchmark also uses 'Proper Case' style

Reply:
Updated